packages/flutter/lib/src/material/popup_menu_theme.dart

```diff
@@ -0,0 +1,164 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show lerpDouble;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+import 'theme.dart';
+
+/// Defines the visual properties of [PopupMenuEntry] widgets.
```

Comment:
I think it would be clearer to say ...properties of the routes used to display popup menus as well as  [PopupMenuItem] and [PopupMenuDivider] widgets.

Reply:
Done